_posts/2019-08-23-unet/unet.Rmd

```diff
@@ -0,0 +1,437 @@
+---
+title: "Image segmentation with U-Net"
+description: > 
+ In image segmentation, every pixel of an image is assigned a class. Depending on the application, classes could be different cell types; or the task could be binary, as in "cancer cell yes or no?". Area of application nonwithstanding, the established neural network architecture of choice is U-Net. In this post, we show how to preprocess data and train a U-Net model on the Kaggle Carvana image segmentation data.
+ 
+author:
+  - name: Daniel Falbel and Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: falbelkeydana2019unet
+date: 08-23-2019
+categories:
+  - Images
+  - Introductions
+  - Keras
+bibliography: bibliography.bib
+output:
+  distill::distill_article:
+    self_contained: false
+preview: images/unet.png
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, eval = FALSE)
+```
+
+Sure, it is nice when I have a picture of some object, and a neural network can tell me what kind of object that is. More realistically, there might be several salient objects in that picture, and it tells me what they are, and where they are. The latter task especially (known as _object detection_) seems prototypical of contemporary AI applications that at the same time are fascinating intellectually and questionable ethically. It's different with the subject of this post: Successful _image segmentation_ has a lot of undeniably useful applications. For example, it is a sine qua non in medicine, neuroscience, biology and other life sciences.
```

Comment:
```suggestion
Sure, it is nice when I have a picture of some object, and a neural network can tell me what kind of object that is. More realistically, there might be several salient objects in that picture, and it tells me what they are, and where they are. The latter task (known as _object detection_) seems especially prototypical of contemporary AI applications that at the same time are intellectually fascinating and ethically questionable. It's different with the subject of this post: Successful _image segmentation_ has a lot of undeniably useful applications. For example, it is a sine qua non in medicine, neuroscience, biology and other life sciences.
```

Reply:
Moved "especially" to be after the parenthetical (since you want the aside to be as close to what it's specifying as possible).

I also changed the order of "intellectually fascinating" (from fascinating intellectually) and "ethically questionable" (same flip-flop). I think it reads a bit more naturally that way, but, as always, feel free to override.

Reply:
> It's different with the subject of this post:

I'm not 100% clear on what "it" refers to here.

Reply:
For the changes, sure, thanks! I had the feeling that some sentences were on the verge of being too complicated ... 
For this
> It's different with the subject of this post:
that was supposed to refer to the "questionnable ethically" thing ...  meaning, for segmentation, one tends to think more of medical applications (supposed to be "good"/useful) than of surveillance (as is the case with object detection) ... If you have an idea how to better convey this, I'd be happy to hear!